openfisca-web-api/deprecated-api.md

```diff
@@ -0,0 +1,639 @@
+# Deprecated API
+
+> This is the documentation for the old OpenFisca web API.  
+> Note that it isn't updated and that its stability isn't guaranteed anymore.  
+> Please use our [last API](http://openfisca.org/doc/openfisca-web-api) instead.  
+
+## Public instance for France
```

Comment:
I would delete this paragraph

Reply:
Ok for deleting it when we stop serving the deprecated api. While it's still active, one who already has the link might look for information about it in the documentation.